frontend/nuxt.config.ts

```diff
@@ -8,17 +8,11 @@ export default defineNuxtConfig({
 		public: {
 			version: process.env.npm_package_version,
 			// URL to use when sending GraphQL queries to the CCDscan API.
-			// (env NUXT_PUBLIC_API_URL)
-			apiUrl: 'http://localhost:5090/graphql',
-			// URL to use when using websockets in GraphQL CCDscan API.
-			// (env NUXT_PUBLIC_WS_URL)
-			wsUrl: 'ws://localhost:5090/graphql',
-			// URL to use when sending GraphQL queries to the CCDscan API.
 			// (env NUXT_PUBLIC_API_URL_RUST)
 			apiUrlRust: 'http://localhost:8000/api/graphql',
 			// URL to use when using websockets in GraphQL CCDscan API.
 			// (env NUXT_PUBLIC_WS_URL_RUST)
-			wsUrlRust: 'ws://localhost:8000/api/graphql',
+			wsUrlRust: 'ws://localhost:8000/ws/graphql',
```

Comment:
```suggestion
			wsUrlRust: 'ws://localhost:8000/api/graphql',
```

Reply:
I think the url for the websocket is correct https://github.com/Concordium/concordium-scan/blob/cd7a0d2a2a73c25483e7d02be2577ee120a4c31a/backend-rust/src/graphql_api.rs#L191

Reply:
Then the env files at the front-end are wrong.